packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme/paymentLightTheme.native.ts

```diff
@@ -0,0 +1,1126 @@
+          hover: globalColors.neutral.blueGrayLight[0],
+          focus: globalColors.neutral.blueGrayLight[0],
+          active: globalColors.neutral.blueGrayLight[0],
+          disabled: globalColors.neutral.blueGrayLight[600],
+        },
+        secondary: {
+          default: globalColors.chromatic.azure[600],
+          hover: globalColors.chromatic.azure[600],
+          focus: globalColors.chromatic.azure[600],
+          active: globalColors.chromatic.azure[600],
+          disabled: globalColors.neutral.blueGrayLight[600],
+        },
+        tertiary: {
+          default: globalColors.neutral.blueGrayLight[1000],
+          hover: globalColors.neutral.blueGrayLight[1000],
+          focus: globalColors.neutral.blueGrayLight[1000],
+          active: globalColors.neutral.blueGrayLight[1000],
+          disabled: globalColors.neutral.blueGrayLight[500],
+        },
+        link: {
+          default: globalColors.chromatic.azure[500],
+          hover: globalColors.chromatic.azure[600],
+          focus: globalColors.chromatic.azure[700],
+          active: globalColors.chromatic.azure[700],
+          disabled: globalColors.neutral.blueGrayLight[400],
+        },
+      },
+      icon: {
+        primary: {
+          default: globalColors.neutral.blueGrayLight[0],
+          hover: globalColors.neutral.blueGrayLight[0],
+          focus: globalColors.neutral.blueGrayLight[0],
+          active: globalColors.neutral.blueGrayLight[0],
+          disabled: globalColors.neutral.blueGrayLight[600],
+        },
+        secondary: {
+          default: globalColors.chromatic.azure[600],
+          hover: globalColors.chromatic.azure[600],
+          focus: globalColors.chromatic.azure[600],
+          active: globalColors.chromatic.azure[600],
+          disabled: globalColors.neutral.blueGrayLight[600],
+        },
+        tertiary: {
+          default: globalColors.neutral.blueGrayLight[1000],
+          hover: globalColors.neutral.blueGrayLight[1000],
+          focus: globalColors.neutral.blueGrayLight[1000],
+          active: globalColors.neutral.blueGrayLight[1000],
+          disabled: globalColors.neutral.blueGrayLight[500],
+        },
+        link: {
+          default: globalColors.chromatic.azure[500],
+          hover: globalColors.chromatic.azure[600],
+          focus: globalColors.chromatic.azure[700],
+          active: globalColors.chromatic.azure[700],
+          disabled: globalColors.neutral.blueGrayLight[400],
+        },
+      },
+    },
+  },
+  border,
+  motion: {
+    delay: {
+      '2xshort': 70,
+      xshort: 120,
+      short: 180,
+      long: 3000,
+      xlong: 5000,
+    },
+    duration: {
+      quick1: 70,
+      quick2: 150,
+      quick3: 200,
+      moderate1: 250,
+      moderate2: 300,
+      gentle1: 400,
+      gentle2: 600,
+      gentle3: 900,
+    },
+    easing: {
+      standard: {
+        //@ts-expect-error Need to figure out a way to have a different Theme type for native
```

Comment:
@kamleshchandnani @anuraghazra having to do this is a big bummer. Since our `Theme` type has this as a `string` which is valid for web but not for native. I could add a `string | EasingFactoryFunction` but that would lead to confusion for consumers. 

The best we could to do is add `string | EasingFactoryFunction` and mention it in the description that it would return `string` on web & `EasingFactoryFunction` on native?

Reply:
Yeah agree on the consumer confusion point, probably we could separate out the Theme types but need to think it through. 

> The best we could to do is add string | EasingFactoryFunction and mention it in the description that it would return string on web & EasingFactoryFunction on native?

Yeah this sounds fine for now.

Reply:
@kamleshchandnani thoughts?

Reply:
It returns an `EasingFunctionFactory` which is `type EasingFunctionFactory = { factory: () => EasingFunction };` and `EasingFunction` is `type EasingFunction = (value: number) => number;`

Reply:
I've changed the type for easing to `string | { factory: () => (value: number) => number };`

Avoiding importing a type from `react-native-reanimated` since this would cause issues on web